src/jobs/models/jobManager.js

```diff
@@ -34,16 +38,35 @@ module.exports.reloadCronJobs = async () => {
     const configData = await configHandler.getConfig();
     try {
         const jobs = await databaseConnector.getJobs(contextId);
-        jobs.forEach(async function (job) {
+        for (const job of jobs) {
             if (job.cron_expression !== null) {
-                addCron(job, job.cron_expression, configData);
+                await addCron(job, job.cron_expression, configData);
```

Comment:
why did you add await? add cron is not an async function

Reply:
done...sorry don't know :)